reducers/contacts.js

```diff
@@ -17,7 +25,7 @@ export default function contacts(state = initialState, action) {
     case "FETCH_CONTACTS_COMPLETE":
       return {
         ...state,
-        data: action.data,
+        data: normalizeContacts(action.data),
```

Comment:
I really like the use of helper functions here to keep the reducer clean

Reply:
thanks!